src/components/Subscribe/Subscribe.tsx

```diff
@@ -63,11 +102,19 @@ const Subscribe = ({
     >
       <h3 className={`${px}-subscribe__title`}>{title}</h3>
       {blurb ? <p className={`${px}-subscribe__blurb`}>{blurb}</p> : null}
+
       <Input
         className={`${px}-subscribe__input`}
         type="email"
+        name="email"
         placeholder={inputPlaceholder}
         labelText={inputLabelText}
+        invalid={isInvalid}
+        invalidText={invalidText}
+        warn={warn}
+        warnText={warnText}
+        required
+        onChange={onInputChange}
```

Comment:
I'm guessing you're wanting to allow consumers to flag the field as invalid as the user is typing, but I don't believe that's what UX is desiring.  Instead they seem to request the error state to only be triggered on Form submission, so I'd guess this would be done in the onSubmit handler of the top-level form element.